src/fparser/two/tests/test_entity_decl.py

```diff
@@ -0,0 +1,83 @@
+# Modified work Copyright (c) 2017-2022 Science and Technology
```

Comment:
Many thanks for adding this file. I have to confess that when I first looked at this I was assuming you were a new contributor from the Met Office otherwise I wouldn't have been quite so stringent in my requests!

Reply:
Please could you move this file into the `tests/fortran2003` directory.